cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -841,6 +843,33 @@ __device__ inline bool maybe_has_nulls(page_state_s* s)
   return run_val != s->col.max_level[lvl];
 }
 
+template <int decode_block_size_t, typename state_buf>
```

Comment:
```suggestion
template <int decode_block_size, typename state_buf>
```
nit: remove `_t` suffix since it's a non-type

Reply:
This kernel uses _t in many places for non-types, to the point where it essentially means "template parameter."  In this PR I'm just going with the status quo.  I agree it should probably be changed, but I don't want to change it in this PR, because it's very pervasive and it will add a lot of noise and want to spread things out.

Reply:
Yes, I noticed this too. Let's leave it as it is then.